Packages/version.d/Kamikaze

```diff
@@ -4,5 +4,5 @@ UMIKAZE_BRANCH="master"
 UMIKAZE_HOME="/usr/src/Umikaze"
 
 # This specifies the Kamikaze Variables
-VERSION="Umikaze 2.2.1-RC1"
+VERSION="To_Be_Determined"
```

Comment:
Wouldn't it make more sense to use the git commit tag?
Something like:
```
VERSION=`git log | head -n1 | awk -F' ' '{print $2}'
```

Reply:
Eventually we want to go to some scheme of that sort. However, we have something of a problem in that presently, this "Umikaze" is both a "product" and a "tool". In the long run, we will need to have multiple "active" instances of built images (stable, release, testing, development, features, etc.) That means that we need to have that instance information come from a source other than the repo of the build tool which should become quite stable.

Reply:
Will we? I mean, we could also use the git branch name + date, unless the git branch is in a specific format (i.e. if branch name is a tag, we only use the branch name, but otherwise it becomes, for instance, `RC-20180120`). The expression for that in Bash should be relatively easy to do:
```
if [[ $UMIKAZE_BRANCH == "\d.+\\.\d.+\\.\d.+*" ]]; then 
    echo "release"
else 
    echo "$UMIKAZE_BRANCH-"`date -u +"%Y%M%d"`
fi
```